src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -192,6 +265,12 @@ const CurationPage = (props: ICurationPageProps) => {
                           fieldKey={`mutations/${mutationIndex}/mutation_effect/description`}
                           inputClass={styles.textarea}
                           label="Description of Evidence"
+                          labelIcon={
```

Comment:
There will be a history icon next to each mutation effect radio button label?

Reply:
Yeah, we should have history for all of them (in theory)

Reply:
Nevermind, the Github diff viewer made it look like you added groupHeader and labelIcon Props. Makes sense now